Chaap/Chaap/App/ChaapApp.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct ChaapApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            MainView()
```

Comment:
이거 ContentView에서 MainView 얹는 식으로 처리할 거라(초기 진입시 닉네임 입력 분기처리도 해야해서요) 그대로 ContentView()로 둬도 될 거 같습니다
```suggestion
            ContentView()
```